mod-pubsub-server/src/test/java/org/folio/rest/impl/SubscribersApiTest.java

```diff
@@ -288,4 +253,14 @@ private EventDescriptor postEventDescriptor(EventDescriptor eventDescriptor) {
     assertThat(postResponse.statusCode(), is(HttpStatus.SC_CREATED));
     return new JsonObject(postResponse.body().asString()).mapTo(EventDescriptor.class);
   }
+
+  private void postDeclareSubscriber(SubscriberDescriptor subscriberDescriptor) {
+    RestAssured.given()
+      .spec(spec)
+      .body(JsonObject.mapFrom(subscriberDescriptor).encode())
+      .when()
+      .post(EVENT_TYPES_PATH + DECLARE_SUBSCRIBER_PATH)
+      .then()
+      .statusCode(anyOf(is(HttpStatus.SC_CREATED), anyOf(is(HttpStatus.SC_INTERNAL_SERVER_ERROR))));
```

Comment:
Can you explain this change? Why did it changed to this from `.statusCode(HttpStatus.SC_CREATED);` ?

Reply:
the same reason as commented above